### PR TITLE
v0.2.0.0 feat(consultant): Your Practice overview + Analytics page, worklist directory, shadcn sidebar + IBM Plex

### PR DESCRIPTION
**Pull request by ashish921998**
## Summary

Reworks the consultant module around the agronomist's real workflow, on top of the supporting UI rebuild already on this branch.

**Overview → "Your Practice" (proactive daily panel)**
- Replaces the chart dashboard with a proactive panel: an impression band ("what needs attention"), a derived-state **Farmers to contact** call list (gone quiet 30+ days · reviewed with no fertilizer plan attached), and a right rail (review-queue pointer, practice snapshot, portfolio nutrient bar). Surfaces who to reach out to — what the reactive review queue is structurally blind to — and self-clears as you act. All derived client-side from existing scoped data; no new RPC.

**Analytics page (new)**
- New `/consultant/analytics` destination (sidebar item between Petiole Review and Team) housing the org-wide charts pulled off the Overview: nutrient status across farms, incoming severity mix, weekly review throughput, review pipeline by status, and recommendation adherence. Page-view instrumented (`consultant_analytics_viewed`) so it can be cut if it goes unused.

**Supporting rebuild (earlier on the branch)**
- Farmer Directory as a dense worklist table (tables over cards).
- Consultant sidebar rebuilt on the shadcn Sidebar primitive (icon-collapse, mobile drawer, ⌘B, section breadcrumb).
- Typography migrated to the IBM Plex superfamily.
- Org-wide dashboard data via two read-only `SECURITY DEFINER` RPCs (`get_org_latest_petiole`, `get_org_followup_adherence`).

## Test Coverage
- **175 tests pass (11 files).** New derived-state logic (gone-quiet thresholding, reviewed-no-plan, call-list ordering, impression-band generation, adherence summary) is covered by unit tests in `consultant-dashboard-metrics.test.ts` — boundary cases, ordering, zero-count suppression, missing-farmer skip. New chart/page components are presentational.

## Pre-Landing Review
- `/review` on the Plan 1 + Plan 2 changes: **0 critical, quality 9.0.** Two fixes applied during review: corrected a misleading RLS docstring on `getPlanTriageIdsByOrg` (the `fertilizer_plans` SELECT RLS is org-wide, not caller-scoped — verified safe as a suppression-set), and made the Overview call list surface a clear error when the farmers query fails instead of a false "all clear" that contradicted the impression band.
- Independent Claude adversarial pass: no correctness bugs (no SQL/race/XSS/div-by-zero). Remaining notes are within documented tolerance (±1 day date drift on the 30-day quiet threshold; `sample_date` can be backdated — both flagged in code).

## Design Review
- Follows DESIGN.md "Agronomic Reads Bench": IBM Plex, slate + sage spine, diverging nutrient scale, color always paired with a text label, mono tabular numbers. `react-doctor` clean on changed scope.

## Plan Completion
- Both plans complete: `docs/plans/consultant-your-practice/01-overview-your-practice.md` (Your Practice) and `02-analytics-page.md` (Analytics). v2 enrichments (per-farmer adherence, nutrient trajectory) explicitly deferred.

## Follow-up fixes (post-review)
- **Farmer Directory column alignment.** Header and data rows were separate grids sharing an `auto`-width Status column that resolved to different widths (the "Unpaid" button is wider than the "STATUS" label), drifting Region / Farms / Status out of alignment. Trailing columns are now fixed-width (Farms 4rem, Status 7rem) so every grid computes identical columns; the row chevron got its own column.
- **Dropped the redundant sidebar "Access Mode" footer label** (role is already in the sidebar header).
- **Scoped global text-selection suppression to interactive chrome** (buttons, nav, touch targets) so report names, emails, and nutrient values stay selectable.

---
*Created with [PostHog Code](https://posthog.com/code?ref=pr)*

## Release Notes v0.2.0.0

- **New Features**
  - Added a proactive “Your Practice” daily panel and an org-wide Consultant Analytics page with multiple charts and page-view tracking.
  - Added an organization branding experience, including logo support, and updated versioning.

- **Improvements**
  - Rebuilt the consultant sidebar (responsive, persistent collapse with keyboard toggle), refreshed navigation, and updated IBM Plex typography.
  - Redesigned Farmer Directory and related triage views into dense worklist tables with improved loading/empty states.

- **Bug Fixes**
  - Fixed nutrient chart lab ordering and improved active navigation highlighting for trailing-slash routes (including sulfur/sulphur aliasing).

- **Documentation / Tests / Chores**
  - Updated “Command Center metrics” documentation and expanded dashboard metrics tests; refreshed design notes.

### CODE REVIEW

File: src/components/consultant/dashboard/CommandCenterDashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    clientsQuery.isPending
+  const callListError = triageQuery.isError || clientsQuery.isError || nutrientQuery.isError
+
+  const coreReady =
+    !triageQuery.isPending &&
+    !nutrientQuery.isPending &&
+    !nutrientQuery.isError &&
+    plansQuery.isSuccess &&
+    !clientsQuery.isPending
+  const allCaughtUp = coreReady && !findingsError && findings.length === 0 && callList.length === 0
```
[the remark this answers is not in the document]
Comment (ashish921998) [reply src/components/consultant/dashboard/CommandCenterDashboard.tsx:127]: Verified against the code — **not a real bug; leaving as-is.**

Whether work *exists* is gated entirely on the finding counts, which never depend on `clientsQuery`:

- `allCaughtUp` requires `findings.length === 0`, and that's only true when there are no open reviews, no gone-quiet farmers, and no reviewed-without-plan.
- `goneQuiet` comes from `nutrientQuery`, `reviewed` from `triageQuery` + `plansQuery`, open-review counts from `triageQuery`. `clientsQuery` only builds `farmIndex`, which supplies *navigation context* to render call-list rows — it never decides if work is present.

So if `clientsQuery` fails and any gone-quiet farmers exist, `goneQuietCount > 0` → the `gone_quiet` finding is present → `findings.length > 0` → `allCaughtUp` is `false`. The band renders the gone-quiet line, and `FarmersToContact` shows its error (because `callListError` already includes `clientsQuery.isError`). The gone-quiet rows being dropped in `buildCallList` (`if (!ref) continue`) can't hide work, since those same farmers are still counted in `findings`. Reviewed-no-plan rows don't even need `farmIndex` — they're pushed from the triage row's own `clientUserId`.

The only case where a clients-only failure yields "All caught up" is when `findings.length === 0` — i.e. genuinely no proactive work — so nothing is hidden. (The directory-load failure is surfaced on the farmer-directory page separately.) This matches the deliberately-scoped failure matrix in the L106-111 comment.

File: src/components/consultant/dashboard/CommandCenterDashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+  )
+
+  // Loading/error roll-ups. Adherence failure degrades gracefully (its finding
+  // just drops out), so it never fails the band. The call list needs both the
+  // nutrient query (to detect gone-quiet farms) and the clients query (to
+  // navigate to them), so a failure of either surfaces as a call-list error
+  // rather than a false "all clear". nutrient failure also blocks coreReady so
+  // a broken gone-quiet signal never collapses into "All caught up".
+  const findingsLoading = triageQuery.isPending || nutrientQuery.isPending || plansQuery.isPending
+  const findingsError = triageQuery.isError
```
[the remark this answers is not in the document]
Comment (ashish921998) [reply src/components/consultant/dashboard/CommandCenterDashboard.tsx:113]: Verified against the code — **not a real bug; the suggested fix would be a regression, so leaving as-is.**

The premise is accurate (`findingsError = triageQuery.isError` only), but the failure is not hidden from the user, and pulling `nutrientQuery.isError` into `findingsError` would hurt:

- On a nutrient-only failure, `goneQuiet` is `[]` → `goneQuietCount` is 0 → the `gone_quiet` finding is suppressed. But the failure is still surfaced: `callListError` includes `nutrientQuery.isError`, so `FarmersToContact` (rendered right below the band) shows its error state, and `coreReady` includes `!nutrientQuery.isError`, so `allCaughtUp` is forced `false` (no false "All caught up").
- If `findingsError` included `nutrientQuery.isError`, a nutrient-only failure would replace the **entire** band with its error banner and **hide the still-valid `open_reviews` finding** from the successful triage query. That's worse than letting one finding drop out.

Letting the gone-quiet finding degrade out of the band while surfacing the nutrient error in `FarmersToContact` is the intended graceful-degradation strategy — the same pattern the L106-111 comment documents for adherence.
